main.py

```diff
@@ -14,11 +17,12 @@
 def run():
     action_inputs = ActionInputs().load_from_environment()
 
-    GithubManager().initialize_github_instance(action_inputs.github_token, ISSUES_PER_PAGE_LIMIT)
-    if action_inputs.is_project_state_mining_enabled:
-        GithubManager().initialize_request_session(action_inputs.github_token)
+    py_github = Github(auth=Auth.Token(token=action_inputs.github_token), per_page=ISSUES_PER_PAGE_LIMIT)
+    py_github_projects = GithubProjects(token=action_inputs.github_token)
```

Comment:
Why have we decided to name github_instance `py_github`? Does it have any extra value knowing it is py?

Reply:
You are right. There is no reason like name conflict to keep these names.

Reply:
Renamed in commit a7771c6b6a6963669fdcd12803ce44009e8d7880.